src/Targets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -31,21 +31,32 @@
       </PropertyPageSchema>
 
      <ProjectCapability Include="VB;Managed"/>
+  </ItemGroup>
+
+  <!-- Targets -->
+
+  <!-- Returns Vbc command-line arguments for the language service -->
+  <Target Name="CompileDesignTime"
+          Returns="@(_CompilerCommandLineArgs)"
+          DependsOnTargets="_CheckCompileDesignTimePrerequisite;Compile"
+          Condition="'$(IsCrossTargetingBuild)' != 'true'"
+        >
+
+    <!-- VB Framework projects needs core library references. In the case of Command line scenario, the vbc injects this reference. -->
+    <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
This check doesn't sound right - we should use the same logic that the legacy LS is using which I believe doesn't check the TFM but instead just checks if FrameworkPathOverride is non null.